src/datagrid/plugins/drag-and-extend.tsx

```diff
@@ -155,10 +155,10 @@ const useOnDragSelection = ({
 
         // This means it's horizontal selection
         if (rowHovered && checkIfWithinRow(cellsSelection, { rowIndex })) {
-            const startColumnIndexOffset = columnIndex > cellsSelection.end.columnIndex ? 1 : -1;
-            const startColumnIndex = cellsSelection.end.columnIndex + startColumnIndexOffset;
-            const startRowIndex = cellsSelection.start.rowIndex;
-            const endRowIndex = cellsSelection.end.rowIndex;
+            const startColumnIndexOffset = columnIndex > cellsSelection.end?.columnIndex ? 1 : -1;
+            const startColumnIndex = cellsSelection.end?.columnIndex + startColumnIndexOffset;
```

Comment:
If undefined, result will be NaN

Reply:
It's fine. I have an if statement below to make sure they exist.
It won't never be undefined anyways. This is just a safe guard

Reply:
Alright bro